utils/build-script-impl

```diff
@@ -1511,15 +1511,7 @@ for host in "${ALL_HOSTS[@]}"; do
 
     for product in "${PRODUCTS[@]}"; do
         # Check if we should perform this action.
-        tmp_product=${product}
-        if [[ ${tmp_product} == "libdispatch_static" ]]; then
-            tmp_product=libdispatch
-            LIBDISPATCH_STATIC_CMAKE_OPTIONS=${LIBDISPATCH_CMAKE_OPTIONS[@]}
```

Comment:
Are we ok with losing this line? I not sure what it was specifically added for but if it is not an issue then this LGTM.

Reply:
It seems to want to replicate the libdispatch options across to the static build.  However, that should happen implicitly further below.